tests/model/test_exif.py

```diff
@@ -1,82 +1,78 @@
+    """
 
-    def test_tiff(self):
-        with Image.open(assets.path_to('SBB0000F29300010000/data/OCR-D-IMG/FILE_0001_IMAGE.tif')) as img:
-            exif = OcrdExif(img)
-        self.assertEqual(exif.width, 2875)
-        self.assertEqual(exif.height, 3749)
-        self.assertEqual(exif.xResolution, 300)
-        self.assertEqual(exif.yResolution, 300)
-        self.assertEqual(exif.resolution, 300)
-        self.assertEqual(exif.compression, 'jpeg')
-        self.assertEqual(exif.photometricInterpretation, 'RGB')
-        self.assertEqual(exif.resolutionUnit, 'inches')
+    with Image.open(assets.path_to(path)) as img:
+        ocrd_exif = OcrdExif(img)
+    assert ocrd_exif.width == width
+    assert ocrd_exif.height == height
+    assert ocrd_exif.xResolution == xResolution
+    assert ocrd_exif.yResolution == yResolution
+    assert ocrd_exif.resolution == resolution
+    assert ocrd_exif.resolutionUnit == resolutionUnit
+    assert ocrd_exif.photometricInterpretation == photometricInterpretation
 
-    def test_png1(self):
-        with Image.open(assets.path_to('kant_aufklaerung_1784-binarized/data/OCR-D-IMG-BIN/BIN_0020.png')) as img:
-            exif = OcrdExif(img)
-        self.assertEqual(exif.width, 1457)
-        self.assertEqual(exif.height, 2084)
-        self.assertEqual(exif.xResolution, 295 if pil_below_83 else 294)
-        self.assertEqual(exif.yResolution, 295 if pil_below_83 else 294)
-        self.assertEqual(exif.resolution, 295 if pil_below_83 else 294)
-        self.assertEqual(exif.resolutionUnit, 'inches')
-        self.assertEqual(exif.compression, None)
-        self.assertEqual(exif.photometricInterpretation, '1')
+    assert ocrd_exif.compression == compression
 
-    def test_png2(self):
-        with Image.open(assets.path_to('scribo-test/data/OCR-D-PRE-BIN-SAUVOLA/OCR-D-PRE-BIN-SAUVOLA_0001-BIN_sauvola.png')) as img:
-            exif = OcrdExif(img)
-        self.assertEqual(exif.width, 2097)
-        self.assertEqual(exif.height, 3062)
-        self.assertEqual(exif.xResolution, 1)
-        self.assertEqual(exif.yResolution, 1)
-        self.assertEqual(exif.resolution, 1)
-        self.assertEqual(exif.photometricInterpretation, '1')
-        self.assertEqual(exif.resolutionUnit, 'inches')
 
-    def test_jpg(self):
-        with Image.open(assets.path_to('leptonica_samples/data/OCR-D-IMG/OCR-D-IMG_1555_007.jpg')) as img:
-            exif = OcrdExif(img)
-        self.assertEqual(exif.width, 944)
-        self.assertEqual(exif.height, 1472)
-        self.assertEqual(exif.xResolution, 1)
-        self.assertEqual(exif.yResolution, 1)
-        self.assertEqual(exif.resolution, 1)
-        self.assertEqual(exif.resolutionUnit, 'inches')
-        self.assertEqual(exif.photometricInterpretation, 'RGB')
-        self.assertEqual(exif.resolutionUnit, 'inches')
+@pytest.mark.skipif(not sys.platform.startswith('linux'), reason="not platform-independent/stable")
+def test_ocrd_exif_serialize_xml():
```

Comment:
IIUC this test is new – could you please elaborate / comment why this only works on Linux?

Reply:
Understood. (Then perhaps just add a `todo` comment?)

Reply:
And anyway, used as example of the skipif-mark

Reply:
I cannot now reproduce the original problem, though I have only tested it in Linux but with different python versions and no issue so far. I think the problem only happened in the CI. However, CI did not run for this pull request, I'll open a new one to verify.

Reply:
https://app.circleci.com/pipelines/github/OCR-D/core/1129/workflows/8969786f-c965-4c0f-bcab-5577839e0883/jobs/3540 no problems there either. I'll remove the `skipif` mark for now (but will definitely use that for tests that **are** platform-dependent in the future).